x-pack/plugins/profiling/common/elasticsearch.ts

```diff
@@ -7,31 +7,31 @@
 
 export interface ProfilingESEvent {
   '@timestamp': string;
-  ContainerName: string;
-  ThreadName: string;
-  Count: number;
-  HostID: string;
-  PodName: string;
-  ProjectID: string;
-  StackTraceID: string;
+  'container.name': string;
+  'process.thread.name': string;
+  'Stacktrace.count': number;
+  'host.id': string;
+  'orchestrator.resource.name.name': string;
```

Comment:
is this a typo?

Reply:
Sorry yes. Pushed the fix.

Reply:
Fixed another typo `StackTrace.id` -> `Stacktrace.id`